src/__tests__/fvApplications.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import { FVSBTApplicationsTable } from '../components/pages/home/fvApplications';
+import { api_link } from '../utils/supabase';
+import axios from 'axios';
+import { act } from 'react-dom/test-utils';
+import { toast } from 'react-toastify';
+import '@testing-library/jest-dom';
+
+jest.mock('axios');
+
+jest.mock('react-toastify', () => ({
+  toast: {
+    success: jest.fn(),
+    error: jest.fn(),
+    info: jest.fn(),
+  },
+}));
```

Comment:
this is repeated in many files. I'm sure there is a way to config `jest` and do it in one place only

Reply:
please add that config / setup module and update all existing test files as well.

Reply:
Can we merge all the tests PR first, since there are I guess 2-3 PRs more about tests
I will create another PR to refactor the tests then, if that's alright?

Reply:
OK, please add a task to the Pivotal to sort this out